plugins/inputs/socket_listener/README.md

```diff
@@ -35,6 +35,15 @@ This is a sample configuration for the plugin.
   ## 0 (default) is unlimited.
   # read_timeout = "30s"
 
+  ## Optional SSL configuration.
+  ## Only applies to stream sockets (e.g. TCP).
+  # ssl_cert = "/etc/telegraf/cert.pem"
+  # ssl_key = "/etc/telegraf/key.pem"
+  ## Enable and require client certificate authentication.
+  # ssl_client_auth = false
+  ## CAs used to verify client certificates.
+  # ssl_ca = ["/etc/telegraf/ca.pem"]
```

Comment:
We have a little bit of prior art on the server side: #3191, I would prefer if we use the same variable names, but let me know if you think that is a problem.  The biggest difference is that client_auth is enabled automatically if there are cacerts specified.

Reply:
Makes sense -- definitely want to stay consistent. The config now matches those variable names.